build/run.js

```diff
@@ -100,6 +100,44 @@ function run_project_manager() {
     })
 }
 
+function make_profiling_feature_string(args) {
+    const profiling_level = args['profiling-level']
+    if (profiling_level === undefined) {
+        return ''
+    }
+
+    const features = []
+
+    if (profiling_level.includes('section')) {
+        features.push('enable-section-profiling')
+    }
+    if (profiling_level.includes('task')) {
+        features.push('enable-task-profiling')
+    }
+    if (profiling_level.includes('detail')) {
+        features.push('enable-detail-profiling')
+    }
+    if (profiling_level.includes('debug')) {
+        features.push('enable-debug-profiling')
```

Comment:
In other file, you are defining section, task, SYMBOL, and debug. Here its detail instead of symbol. This is the problem of defining the same name in different files – could you please refactor the code in such a way that the names are defined only in one place globally, please?

Reply:
Is refactored now.